pkg/util/liveconfig/dev.go

```diff
@@ -13,19 +13,19 @@ import (
 )
 
 const (
-	HIVEENVVARIABLE = "HIVEKUBECONFIGPATH"
```

Comment:
@s-amann this is only for local dev right?

Reply:
@jharrington22 yes, this impl of HiveRestConfig will only be available when `isLocalDevelopmentMode` is set to `true`. Otherwise the HiveRestConfig impl in `pkg/util/liveconfig/hive.go` will be used.